components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -399,8 +404,8 @@ subroutine crm_physics_init(state, pbuf2d, species_class)
       call pbuf_set_field(pbuf2d, pbuf_get_index('RPRDSH')     , 0._r8)
       call pbuf_set_field(pbuf2d, pbuf_get_index('RPRDTOT')    , 0._r8)
       call pbuf_set_field(pbuf2d, pbuf_get_index('NEVAPR_SHCU'), 0._r8)
-      call pbuf_set_field(pbuf2d, pbuf_get_index('PREC_SH')    , 0._r8)
-      call pbuf_set_field(pbuf2d, pbuf_get_index('SNOW_SH')    , 0._r8)
+      call pbuf_set_field(pbuf2d, prec_dp_idx                  , 0._r8)
```

Comment:
Why did you only update a couple of these?

Reply:
Well I already added the indices in this routine, so I figured I'd use them. I thought about trying to remove all the extra pbuf_get_index calls, but I figured it's better to leave that to a dedicated clean up PR.